snippets/ai/README.md

```diff
@@ -0,0 +1,34 @@
+# AI Command Snippets
+
+> [!CAUTION]
+> This is an experimental, early-stage snippet that is not meant for production use. 
+
+This snippet adds a suite of commands accessible with the `ai` command. This includes:
+
+|  |  |  |
+|---------|-------------|---------|
+| `ai.ask` | Ask questions about MongoDB | `ai.ask how do I run queries in mongosh?` |
+| `ai.data` | Generate data-related mongosh commands | `ai.data insert some sample user info` |
+| `ai.query` | Generate a MongoDB query | `ai.query find documents where name = "Ada"` |
+| `ai.aggregate` | Generate a MongoDB aggregation | `ai.aggregate find documents where name = "Ada"` |
```

Comment:
imo `data`, `query`, and `aggregate` should just be 1 command since its all just a type of query